reference-architecture/aws-ansible/playbooks/roles/openshift-emptydir-quota/handlers/main.yaml

```diff
@@ -3,3 +3,10 @@
   service:
     name: atomic-openshift-node
     state: restarted
+  when: deployment_type == "openshift-enterprise"
```

Comment:
I would suggest using a variable for the service name rather than using conditionals in the handler.

Also, we should probably push the emptydir quota stuff down into openshift-ansible rather than doing it externally.